application/src/main/resources/application.yaml

```diff
@@ -0,0 +1,5 @@
+management:
+  endpoints:
+    web:
+      exposure:
+        include: prometheus
```

Comment:
Willen we hier ook meteen de Spring Boot actuators incl `health` aanzetten? Of is dat de default

Reply:
Ja, dat is dus de default